packages/cli/src/ui/utils/CodeColorizer.tsx

```diff
@@ -100,9 +100,10 @@ export function colorizeCode(
   language: string | null,
   availableHeight?: number,
   maxWidth?: number,
+  theme?: Theme,
 ): React.ReactNode {
   const codeToHighlight = code.replace(/\n$/, '');
-  const activeTheme = themeManager.getActiveTheme();
+  const activeTheme = theme || themeManager.getActiveTheme();
```

Comment:
please revert all changes to this file. they are not related to this pull request topic.

Reply:
never mind. now I understand. this change makes sense.